src/sarif_converter.ts

```diff
@@ -69,10 +69,11 @@ const getTools = (analysisResults: IAnalysisResult, suggestions: ISarifSuggestio
         level: severity,
       },
       help: {
+        markdown: suggestion.text,
```

Comment:
there is also `help.text`,
if we set it in `help.markdown` I suggest (🥁 ) we ask the engine to send it as `suggest.markdown` insetead?

Reply:
Since the engine team is going to provide SERIF formatted results, i'd not spend time on talking with them about different names in old proprietary format. Agree?

Reply:
do you think that SARIF directly from the engine will happen soon? I worry that it can take much longer until it actually happens. 
Anyway, I wouldn't block this PR because of this - just do think it's an improvement worth suggesting

Reply:
thank you. If it was just me, i'd do it without hesitation. But since it requires inter-team communication, i'd leave it as is now.